content/meetings/2024-09-23/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 23 September 2024
+geekdocAnchor: false
+weight: -12
+---
+<!--
+SPDX-FileCopyrightText: 2024 PNED G.I.E.
+SPDX-FileContributor: Stichting Health-RI
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+## Agenda
+- Next steps on CKAN developement
+
+## Agreements
+### Technical debts for current MS08 release
+
+- fix support of datetime fields;
+
+### Features, improvements and technical debts scoped into next release
+
+- migration to 2.11;
+- support of ckanext-dcat's new version, that included DCAT-AP 3.0;
+- i18n of labels and content;
+- define how catalogue and publisher will be persisted and displayed;
+- prepare catalogue/dataset onboarding documentation;
+- review current schema.
```

Comment:
I would like to add two things: 
- Check if the DateTime issue still exists.
- This was not discussed during the meeting, but there is an issue with too many labels. It is already in scope here: https://github.com/GenomicDataInfrastructure/gdi-userportal-dataset-discovery-service/issues/113. When there are a large number of datasets, the system breaks.

Reply:
Hey @hcvdwerf thanks for your feedback. I added the datetime thing on current MS08 release. And the second bullet is fixed already.